storage-account.tf

```diff
@@ -10,7 +10,7 @@ locals {
 
   // for each client service two containers are created: one named after the service
   // and another one, named {service_name}-rejected, for storing envelopes rejected by bulk-scan
-  client_service_names = ["bulkscan", "sscs", "divorce", "probate", "finrem", "cmc"]
+  client_service_names = ["bulkscan", "sscs", "divorce", "probate", "finrem", "cmc", "fpla"]
```

Comment:
@nitinprabhu Should this be `publiclaw` instead?

Reply:
Yes that's correct container is publiclaw